src/util/newconfig/ConfigConstraints.hpp

```diff
@@ -156,7 +156,8 @@ class Constraint {
 class PortConstraint final : public Constraint {
 public:
     constexpr ~PortConstraint() override
-    = default;
+    {
```

Comment:
Can we have them all in one line or does clang-format change it to multiline here?

Reply:
clang format changes it to multi-line 😔